lookout/style/typos/analyzer.py

```diff
@@ -1,4 +1,5 @@
 """Identifier typos analyzer."""
+
```

Comment:
we do not put a newline after module docstring.

Reply:
Thank you, fixed it.